Source/GlueGenerator/CSGenerator.cpp

```diff
@@ -294,7 +298,7 @@ void FCSGenerator::ExportEnum(UEnum* Enum, FCSScriptBuilder& Builder)
 
 bool FCSGenerator::CanExportFunction(const UStruct* Struct, const UFunction* Function) const
 {
-	if (Blacklist.HasFunction(Struct, Function) && !Whitelist.HasFunction(Struct, Function) || !ShouldExportFunction(Function))
+	if (Blacklist.HasFunction(Struct, Function) || (!Whitelist.HasFunction(Struct, Function) && !ShouldExportFunction(Function)))
```

Comment:
Note that this is to allow `Whitelist` to take precedence over `ShouldExportFunction` as `FinishAddComponent` is marked with `ScriptNoExport` which was being rejected by `ShouldExportFunction`.